### ISSUE TITLE
Signal crashes with network connection (SQLiteConstraintException: error code 19: UNIQUE constraint failed: recipient.group_id)

### ISSUE THREAD

**Issue by timpauli** [signalapp/Signal-Android#11214]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Signal crashes directly after starting the app when a network connection is available.

### Steps to reproduce
- don't be in airplaine mode
- open Signal
- experience crash 

### Device info
**Device:** Oneplus 3T
**Android version:** Android 10 (LineageOS with microg 17.1)
**Signal version:** 5.7.6

### Link to debug log
sorry, can't post a link to a debug log, because signal directly crashes with a network connection but here is the top line!
is there another way to extract the debug log? can't seem to copy it..
```
SQLiteConstraintException: error code 19: UNIQUE constraint failed: recipient.group_id
```

thanks a lot!

**Comment by hiqua** [comment]
> is there another way to extract the debug log?

You can at least reach the submit debug log window by using your workaround. Then you could either

* search for ` E ` (errors) or / and ` W ` (warnings) in the log and take screenshots, hoping that's enough information
* use `adb logcat` and try to extract the logs from there

That's not optimal, but I don't think there's another way.

**Comment by greyson-signal** [comment]
@timpauli Hi there, I believe you're hitting a crash that is hopefully resolved in 5.8.4, which is currently in beta and slated to start rolling out to prod on Monday. If you'd like to get it early, you can join the [beta channel](https://support.signal.org/hc/en-us/articles/360007318471-Signal-Beta). Thanks!
